addons/website/static/src/scss/website.scss

```diff
@@ -1251,3 +1251,9 @@ table.table_desc tr td {
 .o_perk span {
     display: block;
 }
+
+// safari transform phone number in link black by default (opw-2230301)
+a[href^=tel] {
+ text-decoration:inherit;
+ color: inherit;
```

Comment:
Maybe use `$link-decoration` and `$link-color` instead of inherit? Otherwise my guess is that it will be the same color as text and not as link... could be good enough for a crappy safari fix though?

Actually this could be even better:
```scss
a[href^=tel] {
    @extend a;
}
```
-> force the style of the basic `a` rule (unfortunately not complex rule in this case as the extend rule is an abbreviation of the base one, so `.test > a` won't be extended. But more importantly: this does not create a new css rule. This will output: `a, a[href^=tel] {`

Reply:
after double triple and quadruple check, seem just a classic link, so probably nothing to do, except change link to be in another color than the background ;) so it is a just a problem of customize theme that is already improved in master... I suppose that we will support in 3th tab, the link color between the header and the text xD